modules/apps/journal/journal-test/src/testIntegration/java/com/liferay/journal/web/internal/portlet/action/test/ImportDataDefinitionMVCActionCommandTest.java

```diff
@@ -57,6 +66,65 @@
 	public static final LiferayIntegrationTestRule liferayIntegrationTestRule =
 		new LiferayIntegrationTestRule();
 
+	@Test
+	public void testImportActionGeneratesUniqueFieldNames() throws Exception {
```

Comment:
@rebsilva, can we chat about this pull tomorrow after our daily meeting?

Thanks

Reply:
Sure Renato, thanks!